### PR TITLE
chore: add issue templates

### PR DESCRIPTION
**Pull request by alejandrodnm**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+name: Feature request
+description: Suggest a new feature for TimescaleDB
```
Comment (JamesGuthrie) [inline .github/ISSUE_TEMPLATE/feature_request.yml:3]:
```suggestion
description: Suggest a new feature
```

**Review by JamesGuthrie — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
+---
+name: Bug report
```
Comment (smoya) [inline .github/ISSUE_TEMPLATE/bug_report.yml:2]: Nit to improve UX. Easy to spot in the list.
```suggestion
name: 🐛 Bug report
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  - type: dropdown
+    id: installation
+    attributes:
+      label: What installation method did you use?
+      multiple: true
+      options:
+        - Docker
+        - Source
+        - Pypi
+        - Not applicable
```
Comment (smoya) [inline .github/ISSUE_TEMPLATE/bug_report.yml:74]:
```suggestion
        - Other (Please specify in the description of the bug)
        - Not applicable
```

**Review by JamesGuthrie — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,115 @@
+---
+name: Bug report
+description: Is something not working? Help us fix it!
```
Comment (smoya) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]: Sounds way more collaborative.
```suggestion
description: Is something not working? Let's fix it together!
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
+---
+name: Feature request
```
Comment (smoya) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]: Nit: same as above.
```suggestion
name: 💡 Feature request
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+  - type: textarea
+    id: implementation
+    attributes:
+      label: Implementation challenges
+      description: |
+        If you have ideas of how to implement the feature, and any particularly
+        challenging issues to overcome, then provide them here.
+    validations:
+      required: false
```
Comment (smoya) [inline .github/ISSUE_TEMPLATE/feature_request.yml:44]:
```suggestion
      required: false
  - type: dropdown
    id: work-on-it
    attributes:
      label: Are you going to work on this feature?
      description: No pressure at all, but we’re ready to help you navigate the contribution process.
      options:
        - "🦸 Yes , I will submit a PR soon!"
        - "🆘 No, could someone else please consider working on it?
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+    id: reproduce
+    attributes:
+      label: How can we reproduce the bug?
+      description: |
+        Please try to provide step-by-step instructions how to
+        reproduce the issue. If possible, provide scripts that we can
+        run to trigger the bug.
+      render: bash
+    validations:
+      required: true
```
Comment (smoya) [inline .github/ISSUE_TEMPLATE/bug_report.yml:115]:
```suggestion
      required: true
  - type: dropdown
    id: work-on-it
    attributes:
      label: Are you going to work on the bugfix?
      description: No pressure at all, but we’re ready to help you navigate the contribution process.
      options:
        - "🦸 Yes , I will submit a PR soon!"
        - "🆘 No, could someone else please work on the bugfix?
```

**Review by smoya — COMMENTED** [review]
I left some suggestions

**Pull request merged**